src/verify.py

```diff
@@ -0,0 +1,533 @@
+#!/usr/bin/env python3
+
+"""Generate and run test content"""
```

Comment:
Not sure I get it. Is this a unittest, or a verification script that you run on the post-analysis?

Reply:
If its a unittest, please make sure the file name includes the words "unittest" (e.g. "verify-unittest.py"), and that you add in the commit message an example of how to run it. Also, you may want to add some prefix with more meaning than just "verify".

Reply:
OK. I moved all test related files to a separate fold named "test".
The file is split up into verify_generate.py for the content creation and verify_unittests.py where the tests are much simpler to write:
"
def test10():
    """
    Frozen frames at sync position. No delays and perfect sync.
    """
    audio = 0.0
    video = 0.0
    descr = f"{inspect.currentframe().f_code.co_name}_vd.{video}s.ad.{audio}s"
    # In 60 fps number scheme
    frames = [*range(160, 190)]
    settings = {
        "outfile": f"{descr}.mov",
        "descr": f"{descr}",
        "output_fps": 60,
        "video_delay": video,
        "audio_delay": audio,
        "freeze_frames": frames,
    }

    return run_test(**settings)
"

In verify.py test cases will be automatically populated from the verify_tests.py file and available test can be queried and run separately. Test content can be kept after test run or deleted (default).